doc/source/command_syntax_usage.rst

```diff
@@ -160,24 +160,23 @@ the 'askpass' box in the Vagrantfile.
 JBoss-EAP and -Scan Variants
 ----------------------------
 
-The JBoss EAP facts are divided into two groups. Rho scans for the
-following facts by default:
-
-  - jboss.eap.running-versions
-
-The following facts are available, but Rho will only scan for them if
-requested using the `--facts` option:
-
-  - jboss.eap.installed-versions
-  - jboss.eap.deploy-dates
-
-The `installed-versions` and `deploy-dates` facts are computed by
-scanning the entire host filesystem with the `find` command. This can
-take a large amount of system resources and could potentially have a
-noticeable impact on the performance of running servers. The default
-facts are a good way to get clues about which systems might have JBoss
-installed before performing a resource-intensive scan, or even
-removing the need for a scan entirely.
+The JBoss facts are divided into two groups. The plain jboss-eap facts
```

Comment:
Reading this I understand that I would be able to run `rho scan ... --facts jboss-eap` for the first fact group and `rho scan ... --facts jboss-eap-scan` for the full scan. That said, is that possible? If not I think having example on how to do one or the other scan would be great.

Reply:
No, my mistake. Neither of those are actually fact groups. I took that part out and reworked the documentation to hopefully be more clear. Please let me know what you think!